### DISCUSSION TITLE
Advanced User's Metas - top/list/parse other

### DISCUSSION THREAD

**Discussion by thboehi** [Zrips/CMI#6808] [Enhancement]
Hi everyone!

I've been using a lot `/cmi usermeta`  with Custom Alias to save some data on my players.

It is very helpful and easy to get information back. It still miss some potential like the ability to get for example all player's meta "sponsorship" and make a TOP 3 for example. Or maybe just to be able to get all player's specific meta. For example "`/cmi allusermeta list sponsorship`" and you get all players amount.

I also though about user_meta placeholders. That's sad not being able to specific the player. For example `%cmi_other_WAXIE_metaint_sponsorship%`. That would be great for some occasion.

What do you all think?

(and sorry for my english, if you don't understand, I'll try to use Google Translate to try again...)

**Comment by CloudeLecaw** [comment]
You can use parseothers Expansion of Papi to get placeholders of other players that are online.
